client/reader/following-edit/index.jsx

```diff
@@ -34,7 +34,7 @@ import FollowingImportButton from './import-button';
 import FeedDisplayHelper from 'reader/lib/feed-display-helper';
 import SectionHeader from 'components/section-header';
 import Button from 'components/button';
-const stats = require( 'reader/stats' );
+import * as stats from 'reader/stats';
```

Comment:
No, this change was done manually following the import codemod. 

For the sake of completeness, I'll go ahead replace instances of `import *`s with named imports.